src/util/system.cpp

```diff
@@ -73,6 +73,8 @@
 #include <malloc.h>
 #endif
 
+#include <boost/algorithm/string/replace.hpp>
```

Comment:
Consider adding a project local `ReplaceAll(…)` instead of bringing in `boost::replace_all`.

Reply:
Using dependencies is better than rewriting them.

Reply:
@practicalswift `boost::replace_all` is already a dependency.

Beside that, I'm considering `EncodeBase64` the wallet name/path. WDYT?

Reply:
`replace_all` is already used in four places. I'd suggest to simply use it here, too.
Rewriting it could always be done, it's irrelevant to his PR.